src/InputForm.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function InputForm({ todo, onChange, onCreate }) {
+  return (
+    <div>
+      <input placeholder="할 일을 입력해 주세요" value={todo || ''} onChange={onChange} />

```

Comment:
```suggestion
      <input
        type="text"
        placeholder="할 일을 입력해 주세요"
        value={todo || ''}
        onChange={onChange}
      />
```
type 잊지 말아주세요. 그리고 가독성을 위해 줄 바꿈 해주시면 좋습니다.